graphql/README.md

```diff
@@ -41,24 +39,22 @@ be used.
   Tools](https://www.apollographql.com/docs/react/features/developer-tooling)**
   Chrome Extension offering developer tools for Apollo projects.
 
-## Vocabulary
-
-## Patterns
-
-### Apollo
-
-### Relay
-
 ## Best Practices
 
-## Recommendations
-
-### Error Reporting
-
-### Deleting Resources
-
-## Authn, Authz
-
-## Open Questions
-
-
+* Follow the latest version of the [GraphQL specification].
+* When serving over HTTP, respond with a 200 OK status code to all GraphQL
+  queries.
+* If a client or server error occurs, use the `errors` key in the GraphQL
+  response.
+* If a user-facing error occurs (such as invalid user input), use the `data` key
+  in the GraphQL response.
+* If a mutation can fail because of a user error, use a union type to describe
+  the possible outcomes.
+* If there is an authenticated user, provide the user in the context for the
```

Comment:
Is this an implementation detail?

Reply:
I don't think it is? I can't find a mention of the ctx/context argument in the spec, but it's been in every GraphQL server implementation I've seen. Specifying where authentication lives seems like a useful thing to decide once instead of on every project.